sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -726,6 +726,14 @@ private[sql] class SQLConf extends Serializable with CatalystConf with Logging {
   }
 
   /**
+   * Return the value of an optional Spark SQL configuration property for the given key. If the key
+   * is not set yet, returns None.
+   */
+  def getOptionalConf[T](entry: OptionalConfigEntry[T]): Option[T] = {
```

Comment:
Its used at https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala#L549, so added the require line.